wikihow2zim/entrypoint.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import argparse
+
+from .constants import NAME, setDebug, getLogger
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog=NAME,
+        description="Scraper to create ZIM files from Kolibri channels",
+    )
+
+    parser.add_argument(
+        "--language",
+        choices=['en', 'de', 'fr'],
+        default='en',
+        help="Choose the language (en, de, fr ...)"
+    )
+
+    parser.add_argument(
+        "--optimization-cache",
+        help="URL with credentials to S3 for using as optimization cache",
+        dest="s3_url_with_credentials",
+    )
+
+    parser.add_argument(
+        "--low-quality",
+        help="Uses only the `low_res` version of videos if available. "
```

Comment:
Wikihow only offer a single video version. We'll use this version to decide which encode profile to use so `Use lower-quality, smaller file-size video encode` would be better